app/assets/javascripts/components/coordinates-search-form.js

```diff
@@ -6,7 +6,8 @@
 
   var CoordinatesSearchForm = function(options) {
     this.$container = options.$container;
-    this.$input = options.$input;
+    this.$latitudeInput = options.$latitudeInput;
+    this.$longitudeInput = options.$longitudeInput;
```

Comment:
Taking another look at this after some time away and… I don't think these `<input>`s are necessary…